src/detect-flow.c

```diff
@@ -82,12 +82,11 @@ void DetectFlowRegister (void)
 /**
  * \param pflags packet flags (p->flags)
  * \param pflowflags packet flow flags (p->flowflags)
- * \param tflags detection flags (det_ctx->flags)
  * \param dflags detect flow flags
  * \param match_cnt number of matches to trigger
  */
-static inline int FlowMatch(const uint32_t pflags, const uint8_t pflowflags, const uint16_t tflags,
-        const uint16_t dflags, const uint16_t match_cnt)
+static inline int FlowMatch(const uint32_t pflags, const uint8_t pflowflags, const uint16_t dflags,
```

Comment:
Pre-existing but this function is a candidate for a signature change to return a `bool` instead of an `int`